lib/monitor/monitor.go

```diff
@@ -18,31 +18,51 @@ import (
 var Sink = "0.0.0.0"
```

Comment:
Please add a short overview how the ```Monitor```, ```Stats```, ```Measure``` and ```Measurement``` go together. Btw: what is the difference between ```Measure``` and ```Measurement```?